setup.cfg

```diff
@@ -0,0 +1,66 @@
+[metadata]
+name = SCons
+version=3.9.9a991
+license = MIT
+author = William Deegan
+author_email =bill@baddogconsulting.com
+long_description = file: README.rst
+description = Open Source next-generation build tool.
```

Comment:
Maybe a better description is due?  "next-generation" from a 20-year old?

Reply:
Like Star Trek Next Generation.. that's 30 years ago right? ;)